Demo/Application/Base/AppDelegate.swift

```diff
@@ -28,6 +28,8 @@ import BraintreeCore
             userDefaults.set(BraintreeDemoAuthType.clientToken.rawValue, forKey: BraintreeDemoSettings.AuthorizationTypeDefaultsKey)
         } else if processInfoArgs.contains("-TokenizationKey") {
             userDefaults.set(BraintreeDemoAuthType.tokenizationKey.rawValue, forKey: BraintreeDemoSettings.AuthorizationTypeDefaultsKey)
+        } else if processInfoArgs.contains("-NewPayPalCheckoutTokenizationKey") {
```

Comment:
Can this be removed now? Not sure what it was even added for

Reply:
It sounds like this feature is not enabled for our sandbox tokenization key on iOS. I had asked the same on the Android PR as well.